joomla/joomla-cms
# [J5.3] Add an option to Cassiopeia to remove padding in horizontal menu

Pull Request for Issue #37902.
Cassiopeia template Logo and menu alignment #37902

### Summary of Changes
Added an Option in Cassiopeia Settings to remove the horizontal padding in the menu
If set to yes a class is added into the header
The class is set in metismenu scss to remove the left padding from the first item and the right padding from the last item

### Testing Instructions
In Template Styles of Cassiopeia, set this option to yes
<img width="367" alt="grafik" src="https://github.com/user-attachments/assets/e8ccebac-8c74-46ae-b9fd-756da5350472">


Logo should be left aligned with the menu
<img width="86" alt="grafik" src="https://github.com/user-attachments/assets/d3c92d47-f2e2-40bf-8330-49ae7b721d30">

Set the Option to no

Logo is aligned with the ul List
 
<img width="103" alt="grafik" src="https://github.com/user-attachments/assets/d4253b62-aa25-4187-94b7-a71ca20fbf16">

### Actual result BEFORE applying this Pull Request
Before the patch there is only the option to align to the ul list

### Expected result AFTER applying this Pull Request
Like other templates already have this, I added an additional option to remove the left and right padding.

### Link to documentations
Please select:
- [ x] Documentation link for docs.joomla.org: <link>
_Can someone tell me where to send the documentation to?_


- [ x] No documentation changes for manual.joomla.org needed
